tests/testEqualityConstraint.cpp

```diff
@@ -146,6 +146,25 @@ TEST(EqualityConstraint, VectorExpressionEquality) {
   EXPECT_CORRECT_FACTOR_JACOBIANS(*merit_factor, values2, 1e-7, 1e-5);
 }
 
+TEST(EqualityConstraint, Container) {
```

Comment:
Had not expected this in this PR

Reply:
Since we added EqualityConstraints class, this is the corresponding unit test